packages/gamut-labs/src/brand/Avatar/index.tsx

```diff
@@ -12,7 +12,11 @@ type AvatarImageProps =
   | { alt?: never; 'aria-labelledby': string };
 
 type AvatarBaseProps = {
+  /**path to image asset
+   */
```

Comment:
Just a little comment formattting nit.
```suggestion
  /**
   * path to image asset
   */
```